sycl/doc/extensions/experimental/sycl_ext_oneapi_graph.asciidoc

```diff
@@ -1011,6 +1016,146 @@ If a node containing a dynamic parameter is updated through the whole graph
 update API, then any previous updates to the dynamic parameter will be reflected
 in the new graph.
 
+==== Graph-Owned Memory Allocations [[graph-memory-allocations]]
+
+It can be desirable for a graph to own and manage memory allocations for memory
+associated with commands in the graph. This is made possible by using the
+`async_malloc` and `async_free` functions from the
+link:../experimental/sycl_ext_oneapi_async_memory_alloc.asciidoc[sycl_ext_oneapi_async_memory_alloc
+] extension. These commands can be added to a graph either via queue recording
+or explicit graph creation, which will create allocations which are owned and
+managed by that specific `command_graph`.
+
+Graphs use virtual memory to manage these allocations. To use this feature in a
```

Comment:
I think we should reiterate that the async malloc functions must return a valid USM pointer, but we shouldn't require that this is implemented using virtual memory, this should be an implementation detail.

I think we could keep a note about this, but I'd suggest moving this to an implementation details section, so it's non-normative.

Reply:
Wording about virtual memory has now been removed.